### PR TITLE
fix(examples): correct deep_cfr_pytorch convergence

### LINKED ISSUE #1376: open_spiel/python/examples/deep_cfr_pytorch.py fails to converge
The example given in open_spiel/python/examples/deep_cfr_pytorch.py fails to converge to the known average policy value of 1/18.  It looks like this was tested at one point and converged as expected.

### PR DESCRIPTION
**Pull request by SH20RAJ**
Fixes #1376.

The previous implementation of Deep CFR in PyTorch failed to converge because the advantage network was trained for only 1 step per iteration while being re-initialized. This prevented any meaningful learning.

This PR:
1.  Updates `open_spiel/python/pytorch/deep_cfr.py` defaults to match the TF2 implementation and the paper (more training steps, larger batch sizes).
    - `batch_size_advantage`: 2048 (was None)
    - `batch_size_strategy`: 2048 (was None)
    - `policy_network_train_steps`: 5000 (was 1)
    - `advantage_network_train_steps`: 750 (was 1)
    - `reinitialize_advantage_networks`: True (default retained, but now safe with more steps)
    - `learning_rate`: 1e-3 (was 1e-4) to match TF2.

2.  Updates `open_spiel/python/examples/deep_cfr_pytorch.py` to use these new defaults (removing explicit None overrides).

Tested on `kuhn_poker`, NashConv converged to ~0.05-0.16 (depending on iterations) which is expected for convergence.

### CODE REVIEW

**Comment by alexunderch** [comment]
Duplicate pull request.

**Comment by lanctot** [comment]
Thanks. 

Right, I was going to ask if you saw #1406  ?

**Comment by lanctot** [comment]
Closing as a duplicate of the recent PRs.
